services/api-server/src/simcore_service_api_server/api/routes/solvers_jobs_getters.py

```diff
@@ -22,9 +22,10 @@
 from ...exceptions.custom_errors import InsufficientCreditsError, MissingWalletError
 from ...exceptions.service_errors_utils import DEFAULT_BACKEND_SERVICE_STATUS_CODES
 from ...models.basic_types import LogStreamingResponse, VersionStr
+from ...models.domain.files import File as DomainFile
 from ...models.pagination import Page, PaginationParams
 from ...models.schemas.errors import ErrorGet
-from ...models.schemas.files import File
+from ...models.schemas.files import File as SchemaFile
 from ...models.schemas.jobs import (
```

Comment:
I am still unconfortable with this domain/schema nomenclature.
If I get it right the schemas are actually the ones used with the REST API so why we do not just call them `models.rest.file` ?
the domain ones are the schemas or models used internally by the API-server only, then they should be called as such... eg. `models.internals.file` or just `models.file`. What do you think?